src/dashboards/videos/details.jsonnet

```diff
@@ -264,3 +274,120 @@ dashboard.new(
   ),
   gridPos={ x: 18, y: 0, w: 6, h: 9 }
 )
+.addPanel(
+  {
+    title: 'Event distribution during the video',
+    description: |||
+      We divide the video duration into equal intervals and for each interval display
+      its event distribution.
+      The interval size is controlled by the `Event group interval` variable
+    |||,
+    aliasColors: {},
+    avgLineShow: false,
+    barValuesShow: true,
+    cacheTimeout: null,
+    chartType: 'stacked bar chart',
+    colorSch: [],
+    colorSet: [
+      {
+        '$$hashKey': 'object:204',
+        color: '#1f77b4',
```

Comment:
Chosen colors belongs to the default grafana 8 palette?

Reply:
I think that as it is directly a JSON the color has to be specified

Reply:
I tried to replace the colors with the default grafana palette but for some reason the plugin seems to ignore this configuration. I thought too that it has to be specified, but surprisingly, removing this part didn't change the output, so removed it.